maven/maven.bzl

```diff
@@ -187,12 +223,12 @@ _generate_maven_repository = repository_rule(
 )
 
 # Implementation of the maven_jvm_artifact rule.
-def _maven_jvm_artifact(artifact_spec, name, visibility, deps = [], exports = [],  **kwargs):
+def _maven_jvm_artifact(artifact_spec, name, visibility, deps = [], exports = [], **kwargs):
     artifact = artifacts.annotate(artifacts.parse_spec(artifact_spec))
     maven_target = "@%s//%s:%s" % (artifact.maven_target_name, _DOWNLOAD_PREFIX, artifact.path)
     import_target = artifact.maven_target_name + "_import"
     target_name = name if name else artifact.third_party_target_name
-    exports = exports + deps # A temporary hack since the existing third_party artifacts use exports instead of deps.
+    exports = exports + deps  # A temporary hack since the existing third_party artifacts use exports instead of deps.
```

Comment:
TODO & issue #?

Reply:
Oooh.  that was a bit of internalisms that snuck out. Also only part of this PR by accident of the formatter.  I need to think about this, but it's decidedly not related to this PR.  I'll file an issue.  It doesn't need a TODO because it will have the issue that points to the concern.

Reply:
Filed #31